go/cmd/bpf/bpf.go

```diff
@@ -0,0 +1,61 @@
+package main
+
+import (
+	"fmt"
+	"github.com/pkg/errors"
+	"os"
+
+	usercorn "github.com/lunixbochs/usercorn/go"
+	"github.com/lunixbochs/usercorn/go/cmd"
+	"github.com/lunixbochs/usercorn/go/loader"
+	"github.com/lunixbochs/usercorn/go/models"
+)
+
+func main() {
+	c := cmd.NewUsercornRawCmd()
+	var packet *string
+
+	c.SetupFlags = func() error {
+		// TODO: Default should be ""
+		packet = c.Flags.String("packet", "crackme.pcap", "packet file to run filter against")
+		return nil
+	}
+
+	c.MakeUsercorn = func(filter string) (models.Usercorn, error) {
+		// TODO: Why is packet never parsing out (always the default!)
```

Comment:
Look here: https://github.com/lunixbochs/usercorn/blob/master/go/cmd/cmd.go#L209
Maybe add some debug prints.

Reply:
Yep I've tried. I confirmed it's being called and they are being parsed before reading from it, but no luck. I can dig in more later.